cod/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/from-to-www-redirect: "true"
+  name: cod
+  namespace: w3cie
+spec:
+  rules:
+  - host: mcod.ch.tudelft.nl
```

Comment:
Should this be the host? Should it not be cod.ch.tudelft.nl?

Reply:
The host should be cod.ch.tudelft.nl.

Reply:
So change it to that? :P

Reply:
```suggestion
  - host: cod.ch.tudelft.nl
```